quickstart/app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Category;
+use App\Http\Requests\Category\StoreRequest;
+use App\Http\Requests\Category\UpdateRequest;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $title = __('list').' '.trans_choice('cs',2);
```

Comment:
```suggestion
        $title = __('list') . ' ' . trans_choice('cs', 2);
```
số 2 là sao nhỉ? o_O? nếu là thông số hệ thống thì bỏ vào config cho m nha